src/Enzo/enzo_SolveHydroEquations.cpp

```diff
@@ -135,6 +135,24 @@ int EnzoBlock::SolveHydroEquations
 	    "Grid::SetMinimumSupport() returned ENZO_FAIL");
     }
   }
+
+  /* Set density floor. */
+  const EnzoFluidFloorConfig& fluid_floor_config
+               = fluid_props->fluid_floor_config();
+  enzo_float density_floor = fluid_floor_config.has_density_floor() ?
+                 fluid_floor_config.density() : 0.0;
+
+  if (density_floor > 0.0) {
+    for (int i=0; i<mx*my*mz; i++) {
+       if (density[i] < density_floor) {
```

Comment:
Maybe use std::max here? (I think performance probably doesn't matter because of optimization but it just seems clearer).

Reply:
Change made!